### ISSUE TITLE
GrblSettingsWindow

### ISSUE THREAD

**Issue by deHarro** [martin2250/OpenCNCPilot#44]
Hi Martin!
Super idea that settings window. One had to look up the meaning of those numbers every time when trying to change something, now you can read in plain text what's up with the parameters.

... but I miss the numbers ;-)
Would it be possible to write down the numbers as well, adjacent to the explanations, since we are used to set "$10=0", not knowing, that this parameter is "Status report options".

If you manage to add the dimension (mm, sec, steps, ...) behind the edit boxes, that would be even greater.
Just an idea.

Harald

**Comment by deHarro** [comment]
Hi Martin,
ok, I try to reproduce the error of "issue 45"...
And after the first three clicks I encounter what made me write this "issue 44".
Have a look at this:
![grafik](https://user-images.githubusercontent.com/26298406/35474646-94332170-0391-11e8-82c6-aa2e1ec7d37d.png)
After pulling the Settings Window a little bit larger, I started to edit some entries.
Hitting apply the to be seen error message appears.
Ok, now what caused the error?
In one fast and smooth move I looked up the number in a file I have handy for that and was back to your program in no time... no, I lie, you know what I mean.

The fact that you already provided us with the dimensions in the tool tips at least, I found out later.
Again: I miss the numbers. (This time without smiley)
Harald

(so, now back to the garbeled numbers...)
